src/lib/crowdin.ts

```diff
@@ -88,10 +89,22 @@ export const createFile = async (
 export const updateOrRestoreFile = async (
   branchName: string,
   file: fs.ReadStream
-): Promise<ResponseObject<SourceFilesModel.File>> => {
+): Promise<ResponseObject<SourceFilesModel.File> | CommonErrorResponse> => {
   const storage = await addStorage(file);
   const branches = await listBranches(branchName);
-  const branchId = branches.data[0].data.id;
+  const branchId = branches?.data[0]?.data?.id;
+
+  if (!branchId) {
+    return {
```

Comment:
does it make sense to throw an error instead of returning an error object

Reply:
Yeah wanted to do that initially, but since Crowdin uses its own structures for error-objects and they’re already handled [here}(https://github.com/mollie/crowdin-cli/blob/66abe35cb0d5415a5bc543bd8e6c49057d4fce77/src/download.ts#L48-L50), I’m returning this object instead. I could `throw {...}` the object though.